carma_record/src/CarmaRecordNode.cpp

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (C) 2020-2021 LEIDOS.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+#include <carma_record/CarmaRecordNode.h>
+
+namespace carma_record
+{
+
+CarmaRecordNode::CarmaRecordNode(){};
```

Comment:
This constructor can be reduced to CarmaRecordNode()=default; in the header file and this line can be removed.